[2024-05-19T20:27:11.095+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:27:11.127+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [queued]>
[2024-05-19T20:27:11.146+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [queued]>
[2024-05-19T20:27:11.148+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:27:11.197+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-06-15 00:00:00+00:00
[2024-05-19T20:27:11.216+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=851) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:27:11.229+0000] {standard_task_runner.py:63} INFO - Started process 869 to run task
[2024-05-19T20:27:11.223+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-06-15T00:00:00+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmposc9pvvx']
[2024-05-19T20:27:11.238+0000] {standard_task_runner.py:91} INFO - Job 295: Subtask process_parquet_files
[2024-05-19T20:27:11.380+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:27:11.731+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-15T00:00:00+00:00'
[2024-05-19T20:27:11.811+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:27:12.021+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:27:12.027+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:27:17.651+0000] {logging_mixin.py:188} INFO - No .parquet files found.
[2024-05-19T20:27:17.654+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:27:17.656+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:27:17.692+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-06-15T00:00:00+00:00, execution_date=20230615T000000, start_date=20240519T202711, end_date=20240519T202717
[2024-05-19T20:27:17.720+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:27:17.751+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:27:17.753+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-19T20:40:24.298+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:40:24.334+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [queued]>
[2024-05-19T20:40:24.344+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [queued]>
[2024-05-19T20:40:24.348+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:40:24.369+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-06-15 00:00:00+00:00
[2024-05-19T20:40:24.384+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2599) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:40:24.389+0000] {standard_task_runner.py:63} INFO - Started process 2621 to run task
[2024-05-19T20:40:24.389+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-06-15T00:00:00+00:00', '--job-id', '412', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpr2xzg4uu']
[2024-05-19T20:40:24.398+0000] {standard_task_runner.py:91} INFO - Job 412: Subtask process_parquet_files
[2024-05-19T20:40:24.511+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-15T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:40:24.640+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-15T00:00:00+00:00'
[2024-05-19T20:40:24.643+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:40:24.686+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:40:24.688+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:40:33.603+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:40:33.610+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/silver_s3.py", line 67, in process_parquet_files
    os.remove(file_path)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'
[2024-05-19T20:40:33.641+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-06-15T00:00:00+00:00, execution_date=20230615T000000, start_date=20240519T204024, end_date=20240519T204033
[2024-05-19T20:40:33.670+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 412 for task process_parquet_files ([Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'; 2621)
[2024-05-19T20:40:33.744+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T20:40:33.762+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:40:33.763+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
